pkg/chart/testdata/ill/templates/wordpress.jinja

```diff
@@ -1,6 +1,6 @@
 #helm:generate dm_template
 {% set PROPERTIES = properties or {} %}
-{% set PROJECT = PROPERTIES['project'] or 'dm-k8s-prod' %}
+{% set PROJECT = PROPERTIES['project'] or 'dm-k8s-testing' %}
```

Comment:
git is trying to rebase from initial commit of the helm repository.  It _can_ be rebased, but I would need to resolve the conflicts for 68 commits.  There are some other creative ways to resolve but it would still cost time.